docs/Inputs.md

```diff
@@ -1646,6 +1648,16 @@ The child component receives the following props from `<ReferenceInput>`:
 - `setPagination`: : function to call to update the pagination of the request for possible values
 - `setSort`: function to call to update the sorting of the request for possible values
 
+**Tip** You can make the `getList()` call lazy by using the `isEnabled` prop. This can be useful when using an `AutocompleteInput` on a resource with a lot of data. The following example only starts fetching the options when the query has at least 2 characters:
```

Comment:
I think that 'isEnabled' is too vague a name. ReferenceInput does a lot of things, fetching choices is only one of them. I'd prefer something like 'enableGetChoices'

You should also explain what value this prop expects.

Reply:
Done